core/src/components/alert/alert.tsx

```diff
@@ -406,13 +408,14 @@ export class Alert implements ComponentInterface, OverlayInterface {
                 <textarea
                   placeholder={i.placeholder}
                   value={i.value}
-                  onInput={e => i.value = (e.target as any).value}
                   id={i.id}
-                  disabled={i.disabled}
                   tabIndex={0}
-                  class={{
-                    'alert-input': true,
-                    'alert-input-disabled': i.disabled || false
+                  {...i.attributes as AlertTextareaAttributes}
+                  disabled={i.disabled || (i.attributes && i.attributes.disabled)}
```

Comment:
The `class` and `onInput` explanations make sense. I'm not sure about the `disabled` one though.

For example, if I did:

```javascript
{
  placeholder: 'Outer Placeholder',
  attributes: {
    placeholder: 'Inner Placeholder'
  }
}
```

`'Inner Placeholder'` would win in this case. But if I did:

```javascript
{
  disabled: true,
  attributes: {
    disabled: false
  }
}
```

`true` would win which is not what I would expect. Can we change it so that the `disabled` in `attributes` wins? This would bring it in line with the rest of the values.

Also, you are checking truthiness here which may lead to unexpected results. Even if we did:
```javascript
(i.attributes && i.attributes.disabled) || i.disabled
```

This would simplify down to: `false || true` which would always return `true`. We should probably do this instead:

```javascript
disabled={i.attributes && i.attributes.disabled !== undefined ? i.attributes.disabled : i.disabled}
```